tests/ios/MASVS-CODE/MASTG-TEST-0087.md

```diff
@@ -66,3 +66,4 @@ These checks can be performed dynamically using [objection](../../../Document/0x
 ```bash
 com.yourcompany.PPClient on (iPhone: 13.2.3) [usb] # ios info binary
 Name                  Type     Encrypted    PIE    ARC    Canary    Stack Exec    RootSafe
+```
```

Comment:
Please go back to v1.5.0 and get the missing text from the original test in
Document/0x06i-Testing-Code-Quality-and-Build-Settings.md

Reply:
sure ,will work on it .

Reply:
Thanks a lot @sk3110x1ng!